src/bitcoin-cli.cpp

```diff
@@ -441,6 +441,19 @@ static int CommandLineRPC(int argc, char *argv[])
             method = args[0];
             args.erase(args.begin()); // Remove trailing method name from arguments vector
         }
+        // Get wallet encryption password from command line
+        if (method == "encryptwallet" && !args.size()) {
+            std::string password, verification;
+            fprintf(stdout, "Enter password to encrypt the wallet : ");
+            if(!std::getline(std::cin, password)) {
+                throw std::runtime_error("Failed to read password");
+            }
+            fprintf(stdout, "Re-enter password : ");
+            if(!std::getline(std::cin, verification) || password != verification) {
+                throw std::runtime_error("Passwords do not match");
```

Comment:
perhaps document about the additional throws so user can expect this new behaviour.

Reply:
This is classic behaviour of enter-then-reenter password, where sould I document ?